src/factories/nodeFlowRun.ts

```diff
@@ -81,13 +80,13 @@ export async function flowRunContainerFactory(node: RunGraphNode) {
 
   async function open(): Promise<void> {
     isOpen = true
+    container.addChild(nodesContainer)
```

Comment:
Worth calling out. Since the node is a BoundsContainer if the container for the sub nodes is added it will be part of the bounds even if it is `visible = false` which is what we were doing before. 

So adding and removing the nodes container when opening and closing. I think this is fine because its only happening when a click is triggered when we're rendering the node and its children anyway. If this was happening in a ticker I think this would be a problem. But I think this is fine.

Reply:
We could add a custom flag that BoundsContainer would check to ACTUALLY skip. But that seemed messier than this and this seemed to work well.